ui/site/src/sound.ts

```diff
@@ -1,7 +1,8 @@
 import { storage } from 'common/storage';
 import { isIOS } from 'common/device';
 import { throttle } from 'common/timing';
-import { defined, charToRole } from 'common';
+import { defined } from 'common';
+import { speakable } from 'chess/sanWriter';
```

Comment:
doesn't that make all pages depend on chessops?

Reply:
it shouldn't, because it's not grabbing from the 'chess' barrel like it was before. sanWriter only imports types from chessops.

but it's always possible esbuild has decided to bundle them together (unlikely). i will check

Reply:
ok, chessops is not loaded on site (sanWriter is clean).

i did discover it is still loaded by round/ctrl courtesy of keyboardMove dependency - that could be fixed by separating keyboardMove's load stub from initModule. having them in the same bundle defeats the purpose.

on the other hand - meh, it's 37k and with cache disabled, we are only sending around 230 kb (compressed) over the wire total with rounds - and that includes i18n, fonts, and piece css,

Reply:
it was only one function bringing it into chess barrel. removed that and it's no longer fetched in rounds.

now it's only loaded by analysis, puzzles, insights, learn, nvui, storm, racer, keyboardMove, voice, editor, chart, and dgt.